### PR TITLE
Path alias repository

### PR DESCRIPTION
**Pull request by dbuytaert**
Turns out there is another service ID, `path.alias_repository` that was renamed to `path_alias.repository`.  For some reason, this is not in the Drupal.org change record at https://www.drupal.org/node/3092086.  
However, patch #76 in https://www.drupal.org/project/drupal/issues/3084983 (i.e. https://www.drupal.org/files/issues/2019-11-07/path-alias_module-3084983-76.d9.patch) shows this rename.  I apologize but I have NOT tested this. Merge with caution.

### CODE REVIEW

File: src/Rector/Deprecation/PathAliasRepositoryRector.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+namespace DrupalRector\Rector\Deprecation;
+
+use PhpParser\Node;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+
+/**
+ * Renames deprecated Drupal::services('path.alias_manager') argument.
```
Comment (damontgomery) [inline src/Rector/Deprecation/PathAliasRepositoryRector.php]: Should this say `path.alias_repository`?

**Review by damontgomery — COMMENTED** [review]
Thanks for the addition.

I've left a comment about a copy & paste issue. One of the drawbacks of having each piece be its own file.

To help people see the status of deprecations, we have created an index file, `drupal-rector/deprecation-index.yml`.

This file includes PhpStan errors which are also shown on https://dev.acquia.com/drupal9/deprecation_status/errors. That page uses these errors to indicate if the issues are covered by Drupal Rector.

Do you want to make these updates? I think we have some capacity to review this as well and make the additional updates if that's OK with you.

Thanks again.

PS. I'm realizing I need to update the README with some information on contributions and some of the things we've discussed, because there are a lot of little things to keep track of.

File: src/Rector/Deprecation/PathAliasRepositoryRector.php (hunk 1 above)
Comment (dbuytaert) [reply src/Rector/Deprecation/PathAliasRepositoryRector.php]: Yes, good catch.

**Comment by dbuytaert** [comment]
Feel free to make those edits and move forward.  I'm in back-to-back meetings all day, every day, so my time to contribute rectors is quite limited.  Don't wait on me.  🚀

File: rector_examples/path_alias_manager_updated.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+/**
+ * This demonstrates the deprecated static calls that might be called from procedural code like `.module` files.
+ */
+
+/**
+ * A simple example using the minimum number of arguments.
+ */
+function simple_example() {
+  \Drupal::service('path_alias.manager');
```
Comment (nerdstein) [inline rector_examples/path_alias_manager_updated.php:11]: Is there a convention for this? In my forthcoming blog post, I was recommending putting both examples "needs updated" and "desired updated" in the same file. Is the "_updated" PHP file.

Comment (damontgomery) [reply rector_examples/path_alias_manager_updated.php:11]: @nerdstein, I've been making pairs of files. The `_updated` or `Updated` ones are copy & paste and then run Drupal Rector to see the changes.

I'd like to gather feedback on the whole examples pattern. What is helpful, what isn't? Can we make it easier to develop / read?

I also have a task internally to look at automated testing of the Rector rules which may cover some of this, but isn't 100% overlapping.

Comment (nerdstein) [reply rector_examples/path_alias_manager_updated.php:11]: I'd appreciate two separate directories and some naming conventions tied to the specific rector rules.

**Comment by nerdstein** [comment]
I've done a code review and this code looks really good.

File: rector_examples/path_alias_manager_updated.php (hunk 2 above)
Comment (damontgomery) [reply rector_examples/path_alias_manager_updated.php:11]: Thanks, @nerdstein.

I agree naming conventions would help. The two separate folders is an interesting one as well.

I think documenting naming conventions will make this easier. There are some currently that are mostly followed, but we should take another look.

**Comment by dbuytaert** [comment]
Did the Path Alias rectors not get merged in?

**Comment by damontgomery** [comment]
I see the 5 in this PR.

Today we worked on additions to the examples and the index file. It looks like those haven't been committed and pushed yet.

I expect to have an update tomorrow, but I think we should be able to merge this PR with those updates.

**Comment by damontgomery** [comment]
Sorry, if you were asking about #47, that PR was closed so that we could put everything together in this PR.

**Comment by damontgomery** [comment]
@dbuytaert, thanks again. This code has been merged.

**Comment by dbuytaert** [comment]
Thank you!

**Pull request merged**